From 6e1e86b3d95c0dda5a0e0ffa7b0bfcef16bfc98a Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 20 Jun 2018 10:40:11 +0200
Subject: [PATCH] strace: Check for mpers support

strace enforces mpers support by default (in order to make aware of the
possibility of incorrect non-native personality decoding early). On some
architectures like Aarch64 this is not supported atm, so letting the configure
script to check whether it is available to fix a build.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 package/devel/strace/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/devel/strace/Makefile b/package/devel/strace/Makefile
index 3507ffb..d3c3d14 100644
--- a/package/devel/strace/Makefile
+++ b/package/devel/strace/Makefile
@@ -55,7 +55,7 @@ config PACKAGE_strace_libunwind
 	default n
 endef
 
-CONFIGURE_ARGS += --with-libunwind=$(if $(CONFIG_PACKAGE_strace_libunwind),yes,no)
+CONFIGURE_ARGS += --enable-mpers=check --with-libunwind=$(if $(CONFIG_PACKAGE_strace_libunwind),yes,no)
 MAKE_FLAGS := \
 	CCOPT="$(TARGET_CFLAGS)"
 
-- 
2.17.1

